src/test/resources/clientData.properties

```diff
@@ -0,0 +1,9 @@
+writer.user.name=0oa157tvtugfFXEhU4x7
+writer.user.password=X7eBCXqlFC7x-mjxG5H91IRv_Bqe1oq7ZwXNA8aq
+writer.user.grant.type=client_credentials
+writer.user.scope=write
+
+reader.user.name=0oa157tvtugfFXEhU4x7
```

Comment:
api.zip.code=${base.host}/zipcode
api.expand=${api.zip.code}/expands
localhost:9090/zipcodes/expand

Reply:
fixed

Reply:
Does not work